zipline/assets/roll_finder.py

```diff
@@ -146,12 +145,54 @@ def __init__(self, trading_calendar, asset_finder, session_reader):
         self.session_reader = session_reader
 
     def _active_contract(self, oc, front, back, dt):
-        prev = dt - self.trading_calendar.day
-        front_vol = self.session_reader.get_value(front, prev, 'volume')
-        back_vol = self.session_reader.get_value(back, prev, 'volume')
-        if back_vol > front_vol:
+        """
+        Return the active contract based on the previous trading day's volume.
+
+        In the rare case that a double volume switch occurs we treat the first
+        switch as the roll. Take the following case for example:
+
+        | _____             _____
+        |      \   __      /       <--- 'G'
+        |       \_/__\____/__
```

Comment:
@dmichalowicz could you check in the changes to switch to `+` for the `F` contract?